spec/requests/v0/caregivers_assistance_claims_spec.rb

```diff
@@ -16,29 +16,37 @@
   let(:get_schema) { -> { VetsJsonSchema::SCHEMAS['10-10CG'].clone } }
 
   describe 'POST /v0/caregivers_assistance_claims/download_pdf' do
+    subject do
+      post('/v0/caregivers_assistance_claims/download_pdf', params: body, headers:)
+    end
+
+    before do
+      allow(Flipper).to receive(:enabled?).with(:caregiver1010).and_return(false)
+    end
```

Comment:
recommend wrapping this in a context around the (sole) spec that relies on it, so we're not restubbing it in :67.